packages/skeleton-item/src/SkeletonItem.js

```diff
@@ -16,18 +16,22 @@ export default class SkeletonItem extends Component {
   };
 
   render() {
+    const { height, marginBottom, maxWidth } = this.props;
+
     return (
       <ThemeContext.Consumer>
-        {({ themeClass }) => (
-          <div
-            className={cx("hig__skeleton-item", themeClass)}
-            style={{
-              maxWidth: this.props.maxWidth,
-              marginBottom: this.props.marginBottom,
-              height: this.props.height
-            }}
-          />
-        )}
+        {({ themeData }) => {
+          const styles = stylesheet(
+            {
+              maxWidth,
+              marginBottom,
+              height
+            },
+            themeData
+          );
+          const classNames = `hig__skeleton-itemV1 ${css(styles.skeletonItem)}`;
```

Comment:
i found this out working on the themeable `spacer` but you can import `cx` from `emotion` to combine classes as well.